### PR TITLE
Ivan how to clone project

### PR DESCRIPTION
**Pull request by workivan**
https://docs.google.com/spreadsheets/d/1ShYjUBfdLCB3lEUn945dh4_t4F0hpbouD-ke6-rfDbc/edit#gid=0

№21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
 
 Это рабочий репозиторий для игры, которую я хочу создать. Пока у нее нет названия, так что это просто игра без имени 
+
+
+###How to clone project
+
+To clone run:
+```shell
+$ git clone https://github.com/gwenbleyd/Untitled-game.git -b development
+```
```
Comment (MaxYarn) [inline README.md]: add in this line, Vanya
To enter in Matrix:
$ cd Untitled-game

Comment (workivan) [reply README.md]: @MaxYarn , matrix ?? 
yes, i add

File: README.md [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+To create new branch, in our case we should mention who's developing in that branch:
+```
+$ git branch branch-name
+branch name example:
+Ivan-write-how-to-clone-project
+```
+To commit code:
```
Comment (MaxYarn) [inline README.md]: Before it, we need to switch branches
To change branch:
$ git checkout name-of-your-branch

File: README.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+``` 
+$ git add EDITEDNAHUECEOFILE — git noted just file you have mentioned
```
Comment (MaxYarn) [inline README.md]: Sometimes we need to indicate file extansion

File: README.md [hunk 4]
```diff
@@ -1,3 +1,30 @@
...
+$ git add. — git noted all files where you have change code or added new one
+
+$ git commit -m "NORMALNO BLYAD EBANYL COD, INDYSU V AHUE"
+```
+To push code:
+```
+$ git push
```
Comment (MaxYarn) [inline README.md]: full version: git push origin your-branch-name

**Pull request merged**